slides/04/04.md

```diff
@@ -616,7 +616,7 @@ to make progress (improve the individual features).
 section: UniversalApproximation
 # Universal Approximation Theorem '89
 
-Let $φ(x):ℝ → ℝ$ be a nonconstant, bounded and nondecreasing continuous function. \
+Let $φ(x):ℝ → ℝ$ be a nonconstant, bounded and non-decreasing continuous function. \
```

Comment:
According to https://www.merriam-webster.com/dictionary/nondecreasing, https://dictionary.cambridge.org/dictionary/english/nondecreasingand for example https://mathworld.wolfram.com/NondecreasingFunction.html it seems fine (in a sense that both are possible); the PRML book that I am referring to quite a lot uses "nonincreasing", so I will stay with "nondecreasing"; will change it after merging.